src/preamble.tex

```diff
@@ -6,3 +6,5 @@
 \usepgfplotslibrary{polar}
 \usetikzlibrary{arrows}
 \tikzset{>=stealth'}
+
+\usetikzlibrary{pgfplots.smithchart}
```

Comment:
Shouldn't this be `\usepgfplotslibrary{smithchart}` to be consistent with how we did polar?

Reply:
Fixed!